File: ch10/spring-boot-example/src/main/java/com/packtpub/hands/on/reactive/kotlin/chapter10/springbootexample/CustomerService.java
package com.packtpub.hands.on.reactive.kotlin.chapter10.springbootexample;

import java.util.Arrays;
import java.util.List;

public class CustomerService implements CustomerInterface {
  private final static List<Customer> customers = Arrays.asList(
      new Customer("Peter", "Parker"),
      new Customer("Steve", "Rogers"),
      new Customer("Red", "Richards"));

  @Override
  public Customer getCustomer(int number) {
    return customers.get(number);
  }
}

File: ch11/spring-boot-kotlin-example/src/main/kotlin/com/packtpub/hands/on/reactive/kotlin/chapter11/springbootkotlinexample/CustomerService.kt
package com.packtpub.hands.on.reactive.kotlin.chapter11.springbootkotlinexample

class CustomerService : CustomerInterface {
  companion object {
    val customers = hashMapOf(
        1 to Customer("Peter", "Parker"),
        2 to Customer("Steve", "Rogers"),
        3 to Customer("Red", "Richards")
    )
  }
  override fun getCustomer(id: Int) = customers[id]
}
